src/main/java/com/maybe/maybe/service/EmployeeService.java

```diff
@@ -14,8 +20,62 @@ public EmployeeService(EmployeeRepository employeeRepository) {
         this.employeeRepository = employeeRepository;
     }
 
-    public Employee findById(Long id) {
-        return employeeRepository.findById(id)
-                .orElseThrow(() -> new EntityNotFoundException("Could not find employee id=" + id));
+    public Employee getEmployeeById(Long id) {
```

Comment:
Don't agree. To my mind, my version of this method is more correct.
Why did you recreate the method getEmployeeById?
It is clear that it returns Employee beasause it is in EmployeeService.